numpy/add_newdocs.py

```diff
@@ -1453,7 +1453,7 @@ def luf(lamdaexpr, *args, **kwargs):
         When True, yield `x`, otherwise yield `y`.
     x, y : array_like, optional
         Values from which to choose. `x` and `y` need to have the same
-        shape as `condition`.
+        shape as `condition`, or be broadcastable to that shape.
```

Comment:
That's also wrong though, for x, y, condition with shapes `(3, 1, 1), (1, 3, 1), (1, 1, 3)`. `(3, 1, 1), (1, 3, 1)` is not broadcastable **to** `(1, 1, 3)`, but it is broadcastable **with** `(1, 1, 3)`